packages/trader/src/Modules/CFD/Containers/cfd-password-modal.jsx

```diff
@@ -307,6 +307,30 @@ const CFDPasswordForm = props => {
     );
 };
 
+const CFDServerModalWarning = ({ show_warning = true, platform }) => {
+    if (!show_warning) return null;
+    return (
+        <div className='cfd-password-modal__warning'>
+            <Text
```

Comment:
`<Text` itself accepts className, possible to remove wrapping div?

Reply:
thanks
I know but I needed that wrapper div for styling purposes
let's keep it